Library/Formula/influxdb.rb

```diff
@@ -91,7 +96,9 @@ def install
 
     cd influxdb_path do
       if build.head?
```

Comment:
If you want this to apply to `devel` as well as `head`, this should be `unless build.stable?`